test_task1.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+import task1
+
+
+class StringManipulation(unittest.TestCase):
+    def test_remove_all_except_first(self):
+        result = task1.remove_all_except_first("TechCity")
+        self.assertEqual(result, "Techiy")
```

Comment:
You may add more strings to test method. E.g; a string with only one character, a string with just repeated characters. Then loop over strings and pass them through the function for assertion